sources/freshdesk/helpers.py

```diff
@@ -0,0 +1,72 @@
+import base64
+from typing import Any, Dict, Iterable
+
+from dlt.sources.helpers import requests
+
+
+def create_auth_headers(api_secret_key: str) -> Dict[str, str]:
+    """
+    Constructs Basic Authorization header required by Freshdesk.
+
+    Args:
+        api_secret_key (str): Freshdesk API secret key.
+
+    Returns:
+        Dict[str, str]: Headers with Basic Authorization.
+    """
+    encoded_key = base64.b64encode(f"{api_secret_key}:X".encode()).decode()
```

Comment:
why you encode it first and then decode? looks like it does nothing, no?

Reply:
yes, corrected it.